12.0/base_requirements.txt

```diff
@@ -81,6 +81,6 @@ ptyprocess==0.5.2
 py==1.5.3
 pycodestyle==2.3.1
 pyflakes==1.6.0
-PyYAML>=4.2b1
+PyYAML==4.2b4
```

Comment:
It is worth to test the 5.1 release on v12, as Odoo doesn't use PyYaml anymore we have no more conflict on this.

Reply:
i guess we could but i think there is also value in having the same version in all templates so you don't have to deal with different bugs. @guewen what do you think.

Reply:
I'd prefer to keep older versions on the older odoo versions but upgrade to newer versions as we move to more recent odoo versions. Make sense to me to try 5.1 on 12.0

Reply:
Is it still relevant to have this lib in v12 as Odoo discontinued the usage of Yaml files ?

Reply:
@p-tombez it is a dependency of marabunta so yes

Reply:
> Is it still relevant to have this lib in v12 as Odoo discontinued the usage of Yaml files ?

No. Probably not much used in OCA addons neither. 

In marabunta, we move to ruamel.yaml and in any case it pulls the dependencies it needs.

→ to remove

Reply:
Didn't know about ruamel, I said nothing then :)

Reply:
dependency dropped for v12